index.js

```diff
@@ -7,12 +7,42 @@ function ask(questionText) {
   });
 }
 
-start();
+//The computer then guesses within the predetermined range
+let computerGuess = (max, min) => {
+  let guess = (min + max) / 2;
+  return Math.floor(guess);
+};
 
+start();
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  let playing = true;
+  let upperBound = 100
```

Comment:
You can use either process.argv or await ask(...) to get user input for the initial max value. If you decide to use await ask(...) remember that it returns a string value.